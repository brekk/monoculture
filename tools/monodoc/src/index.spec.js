import * as MONODOC from './index'

test('default export', () => {
  expect(MONODOC).toMatchSnapshot()
})
