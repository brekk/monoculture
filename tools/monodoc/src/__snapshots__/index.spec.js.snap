// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default export 1`] = `
{
  "addLineNumbers": [Function],
  "cleanupKeywords": [Function],
  "findJSDocKeywords": [Function],
  "formatComment": [Function],
  "getExample": [Function],
  "groupContiguousBlocks": [Function],
  "handleSpecificKeywords": [Function],
  "isJSDocComment": [Function],
  "j2": [Function],
  "lines": [Function],
  "matchLinks": [Function],
  "nixKeyword": [Function],
  "objectifyComments": [Function],
  "parse": [Function],
  "parseFile": [Function],
  "slug": [Function],
  "stripLeadingHyphen": [Function],
  "stripRelative": [Function],
  "structureKeywords": [Function],
  "trimComment": [Function],
  "trimSummary": [Function],
  "unlines": [Function],
  "wipeComment": [Function],
}
`;
